### DISCUSSION TITLE
提示一下各位绳匠：

### DISCUSSION THREAD

**Discussion by HifumiMizuhara** [share121/inter-knot#2227] [其他]
看到很多废了的图片链接
GitHub在上传图片时需要等待：
<img width="1440" alt="スクリーンショット 2024-08-15 17 45 40" src="https://github.com/user-attachments/assets/fafc2738-c79c-489a-a709-dcc576e4a337">
显示“img”打头的才是上传成功了，下面的以“感叹号”结尾且括号内为空的就是上传失败的

**Comment by poolofwater** [comment]
感谢佬的提醒<img src="https://bbs-static.miyoushe.com/static/2024/07/31/213614cd6275ec54d80fe9f84a0edf62_6334766189957723007.png" alt="白露 灭火" width="75" >
